### PR TITLE
Handle the key-up event in text_input_channel

### PR DESCRIPTION
**Pull request by bbrto21**
* Long press event was recently added to the input panel of VD.
  So we have to handle the key-up event appropriately.

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Comment by bbrto21** [comment]
related issue: https://github.com/flutter-tizen/flutter-tizen/issues/327

**Review by JSUYA — APPROVED** [review]
LGTM

**Review by swift-kim — APPROVED** [review]

File: shell/platform/tizen/channels/text_input_channel.cc [hunk 1]
```diff
@@ -124,12 +124,14 @@ TextInputChannel::TextInputChannel(
...
 bool TextInputChannel::SendKeyEvent(Ecore_Event_Key* key, bool is_down) {
-  if (!active_model_ || !is_down) {
+  if (!active_model_) {
     return false;
   }
 
-  if (!FilterEvent(key)) {
-    HandleUnfilteredEvent(key);
+  if (!FilterEvent(key, is_down)) {
+    if (is_down) {
```
[the remark this answers is not in the document]
Comment (bbrto21) [reply shell/platform/tizen/channels/text_input_channel.cc]: Please review again.

**Comment by swift-kim** [comment]
Sorry, the default branch is now flutter-2.10.1-tizen. Please rebase and update the base branch.

**Comment by bbrto21** [comment]
> Sorry, the default branch is now flutter-2.10.1-tizen. Please rebase and update the base branch.

Done!

**Pull request merged**